src/Common/Utils.fs

```diff
@@ -3,8 +3,15 @@ module Common.Utils
 open Microsoft.FSharp.Collections
 open Microsoft.FSharp.Reflection
 
+let private getUnionCase<'a> case = FSharpValue.MakeUnion(case,[||]) :?> 'a
```

Comment:
I would name it create or make union case to keep the same naming as in FSharpValue

Reply:
Good point, renamed